### ISSUE TITLE
Diplomacy is not really explained properly in the game manual, also difference between Normal and Slay rules is not clear.

### ISSUE THREAD

**Issue by MayankGoel28** [yiotro/Antiyoy#37]

**Comment by yiotro** [comment]
Yeah, you're right.

I plan to add little description of diplomacy in next major update. 

About difference between rules: normal rules may change in future (probably I won't do that, but it's possible) and in this case I will need to change text that describes those differences, which may cause problems with some translations.

**Comment by Libervurto** [comment]
There is diplomacy?

**Comment by yiotro** [comment]
> There is diplomacy?

Yes, I've added it with last update. You can enable it in 'skirmish - more'. Be sure to have latest version.

**Comment by Libervurto** [comment]
Yes, I found it.  Nice work.  Makes for an interesting new style of play.

PS: why in the new update does the "End Turn" button swap sides when a territory is selected?  That keeps throwing me off.

**Comment by yiotro** [comment]
> PS: why in the new update does the "End Turn" button swap sides when a territory is selected? That keeps throwing me off.

Hmm, are you talking about new UI for construction? You can actually change that by enabling/disabling 'left handed mode' in game options.

**Comment by Tipit84** [comment]
I like the new Diplomatic mode. The only this is I'd like a little icon showing what countries are at war with each other/allies. And then there was one game where I had the most territory and lost. It was a bit confusing.

**Comment by yiotro** [comment]
> I like the new Diplomatic mode. The only this is I'd like a little icon showing what countries are at war with each other/allies.

So if there are 7 players on field then there will be 5 or 6 icons near each country?

> And then there was one game where I had the most territory and lost. It was a bit confusing.

Maybe he just was able to make friends with everyone faster than you?

**Comment by Tipit84** [comment]
Maybe something like this
![diplomacy](https://user-images.githubusercontent.com/35352189/34895634-a9fbe638-f7b4-11e7-9bb2-6a28dc6d7171.PNG)

**Comment by yiotro** [comment]
> Maybe something like this

Hmm, it looks very confusing to me.
